projects/canopy/src/lib/hero-img/hero-img-card-title/hero-img-card-title.component.scss

```diff
@@ -0,0 +1,17 @@
+@import '../../../styles/mixins.scss';
+
+.lg-hero-img-card-title__heading {
+  > h1,
+  > h2,
+  > h3,
+  > h4,
+  > h5,
+  > h6 {
+    @include lg-font-size('7', 'strong');
+
+    color: var(--color-white);
+    font-family: var(--font-family-expressive);
+    line-height: var(--text-fs-7-line-height);
```

Comment:
The line height should be set by the mixin included above so I think you can delete this line.